servers/cromwell/jobs/controllers/capabilities_controller.py

```diff
@@ -23,3 +25,12 @@ def get_capabilities():
             'cromwell-workflow-name', 'cromwell-workflow-id', 'comment'
         ],
         query_extensions=[])
+    if current_app.config['use_caas']:
+        capabilities.authentication = AuthenticationCapability(
+            is_required=True,
+            scopes=[
+                'profile', 'email', 'openid',
```

Comment:
How did you determine what scopes to use here? It looks pretty different from the scopes used on the dsub side, which is a bit surprising to me because both dsub and cromwell ultimately interact with GCP via Pipelines API.

Reply:
Thanks for pointing that out Dave! Looking at this again, I believe we want to use these scopes: https://github.com/broadinstitute/firecloud-tools/blob/740ea4289a2234ac7a232704f0b4c4d00ba7a5b2/register_service_account/register_service_account.py#L18